### PR TITLE
Tweak timeline card layout

### PR DESCRIPTION
**Pull request by jryans**
This improves the layout of maximised widget chat timelines, including reactions, typing notifications, and other misc. details.

Fixes https://github.com/vector-im/element-web/issues/20846

image

---
Here's what your changelog entry will look like:

## 🐛 Bug Fixes
 * Tweak timeline card layout ([\#7743](https://github.com/matrix-org/matrix-react-sdk/pull/7743)). Fixes vector-im/element-web#20846.

Preview: https://62026b1f8d88e2b7783e7f45--matrix-react-sdk.netlify.app
⚠️ Do you trust the author of this PR? Maybe this build will steal your keys or give you malware. Exercise caution. Use test accounts.

### CODE REVIEW

File: res/css/views/right_panel/_TimelineCard.scss [hunk 1]
```diff
@@ -70,18 +83,32 @@ limitations under the License.
...
     .mx_EventTile:not([data-layout="bubble"]).mx_EventTile_info .mx_EventTile_avatar {
         left: 18px;
     }
+
+    .mx_WhoIsTypingTile {
+        margin-left: -12px;
+    }
+
+    .mx_WhoIsTypingTile_avatars {
+        flex-basis: 48px; // 12 + 36
```
Comment (t3chguy) [inline res/css/views/right_panel/_TimelineCard.scss:112]: could this comment get a bit more detail as to what 12 and 36 represent

File: res/css/views/rooms/_WhoIsTypingTile.scss [hunk 2]
```diff
@@ -23,7 +23,7 @@ limitations under the License.
 
 /* position the indicator in the same place horizontally as .mx_EventTile_avatar. */
 .mx_WhoIsTypingTile_avatars {
-    flex: 0 0 83px; // 18 + 65
+    flex: 0 0 82px; // 18 + 64
```
Comment (t3chguy) [inline res/css/views/rooms/_WhoIsTypingTile.scss:26]: likewise here

Comment (t3chguy) [reply res/css/views/rooms/_WhoIsTypingTile.scss:26]: Also this seems to affect more than just the timeline card, is that intentional and tested?

**Review by t3chguy — APPROVED** [review]
LGTM otherwise

File: res/css/views/rooms/_WhoIsTypingTile.scss (hunk 2 above)
Comment (jryans) [reply res/css/views/rooms/_WhoIsTypingTile.scss:26]: Yes, it's intentional, I noticed while working on this that the typing tile in the main timeline was 1px out of alignment.

**Pull request merged**